railties/lib/rails/generators/rails/plugin/templates/bin/rails.tt

```diff
@@ -1,4 +1,5 @@
-# This command will automatically be run when you run "rails" with Rails 4 gems installed from the root of your application.
+# This command will automatically be run when you run "rails" with Rails 5 gems
```

Comment:
Looking at the history, it was updated before also to change the version https://github.com/rails/rails/commit/2a57cbc7